rootfs/builder/build.sh

```diff
@@ -122,7 +122,7 @@ fi
 
 ## Buildpack detection
 
-buildpacks=("$buildpack_root/*")
+buildpacks=($buildpack_root/*)
```

Comment:
Let's see if the linting will pass with this:

```
buildpacks=(${buildpack_root}/*)
```

Reply:
according to this: https://www.shellcheck.net/ it won't pass:

![image](https://user-images.githubusercontent.com/800750/101885492-77739d00-3b9a-11eb-9f7a-c428158522aa.png)

but we can ignore this check by adding a comment:

![image](https://user-images.githubusercontent.com/800750/101885651-b3a6fd80-3b9a-11eb-91a5-b297dcfcab0d.png)

Reply:
Okay, perfect @n0n0x ! Can you add the comment and rerun the CI and will merge as soon as it passes! 👍

Reply:
pushed!

Reply:
TY @n0n0x for this fix! Waiting for CI to pass.

Going to include this in the next release. It has been buggy for users.

Reply:
Hi @n0n0x , build failed. I think due to misspelling `spellcheck` instead of `shellcheck` of the comment. Can you please fix real quick when you can? :)